typescript/src/client/evm/evm.ts

```diff
@@ -603,4 +606,62 @@ export class EvmClient implements EvmClientInterface {
       ...options,
     });
   }
+
+  /**
+   * Lists CDP EVM token balances.
+   *
+   * @param {ListTokenBalancesOptions} options - Parameters for listing the token balances.
+   * @param {number} [options.pageSize] - The number of token balances to return.
+   * @param {string} [options.pageToken] - The page token to begin listing from.
+   * This is obtained by previous calls to this method.
+   *
+   * @returns A promise that resolves to an array of token balances, and a token to paginate through the token balances.
+   *
+   * @example
+   * ```ts
+   * const tokenBalances = await cdp.evm.listTokenBalances({
+   *   address: "0x1234567890123456789012345678901234567890",
+   *   network: "base-sepolia",
+   * });
+   * ```
+   *
+   * @example
+   * **With pagination**
+   * ```ts
+   * let page = await cdp.evm.listTokenBalances({
+   *   address: "0x1234567890123456789012345678901234567890",
+   *   network: "base-sepolia",
+   * });
+   *
+   * while (page.nextPageToken) {
+   *   page = await cdp.evm.listTokenBalances({
+   *     address: "0x1234567890123456789012345678901234567890",
+   *     network: "base-sepolia",
+   *     pageToken: page.nextPageToken,
+   *   });
+   */
+  async listTokenBalances(options: ListTokenBalancesOptions): Promise<ListTokenBalancesResult> {
```

Comment:
Could you also add a usage example, in `typescript/src/examples/evm`

Reply:
Done.